documentation/milestones/individual_milestones/get_opportunities.md

```diff
@@ -0,0 +1,212 @@
+  - [ ] The endpoint returns all of the grant opportunities that are available on grants.gov
+  - [ ] The endpoint returns at least one (1) field per opportunity
+  - [ ] Updates to the data in legacy Grants.gov are propagated to the new endpoint within 1 hour
+
+### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+1. Number of unique users accessing API
+2. Total number of API calls made
+3. Error rate of API calls
+4. Uptime of service
+5. Deployment/hosting costs
+6. Average response time
+
+### Destination for live updating metrics
+<!-- Required -->
+
+Page on the public wiki that is updated at the end of each sprint. **Note:** This will likely change once we deliver [the Public Measurement Dashboard milestone](../milestone_short_descriptions.md#public-measurement-dashboards)
+
+## Planning
+
+### Assumptions & dependencies
+<!-- Required -->
+
+*What capabilities / milestones do we expect to be in place at the beginning of work on this milestone?*
+
+- [ ] **[API Planning](https://github.com/HHS/grants-equity/issues/):** Determines the language, framework, and deployment service used to build and host the API.
+- [ ] **[DB planning](https://github.com/HHS/grants-equity/issues/):** Determines the DMBS and hosting service used to store and manage the data serviced by the API.
+- [ ] **[Developer tools](https://github.com/HHS/grants-equity/issues/):** Establishes a suite of tools used to ensure the quality and security of the API codebase.
+- [ ] **[beta.grants.gov domain](https://github.com/HHS/grants-equity/issues/):** Secures access to the `beta.grants.gov` domain from which the API endpoints will be routed. 
+- [ ] **[Back-end CI/CD](https://github.com/HHS/grants-equity/issues/):** Sets up a CI/CD pipeline that will be used to test and publish code changes to the API.
+- [ ] **[Data architecture](https://github.com/HHS/grants-equity/issues/):** Establishes the updated data model used to support the new GET Opportunities endpoint.
+- [ ] **[Test data and schema](https://github.com/HHS/grants-equity/issues/):** Enables both project maintainers and open source contributors to effectively mock the database when developing or testing locally.
+- [ ] **[Database (DB) replica](https://github.com/HHS/grants-equity/issues/):** Ensures parity between the set of opportunities returned by the new GET Opportunities endpoint and the legacy system. It also allows users to access the endpoint when there are outages on the legacy system.
+- [ ] **[Feature flag framework](https://github.com/HHS/grants-equity/issues/):** Enables us to deploy new features or changes without immediately exposing them to the public. 
+- [ ] **[API documentation](https://github.com/HHS/grants-equity/issues/):** Establishes a location and strategy for publishing information about the GET Opportunities endpoint (and future API endpoints) that users can reference when learning how to interact with the API.
```

Comment:
Not a blocking comment but there are a few bullets here that we know will probably not be done before this work is started. This is under "Assumptions" so not a big deal!

Reply:
Good call, let me adjust the language for this from "started" to "completed" because that's really what this section has become a record of

Reply:
Made the change in [this commit](https://github.com/HHS/grants-equity/pull/249/commits/93bf2532781b0d4e4820f0d98906ba37713199bb)